homeassistant/components/tesla/sensor.py

```diff
@@ -48,6 +48,11 @@ def __init__(self, tesla_device, controller, sensor_type=None):
         else:
             self.entity_id = ENTITY_ID_FORMAT.format(self.tesla_id)
 
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID."""
+        return self.tesla_id
```

Comment:
Here we need to add `self.type` to the `tesla_id` to create the unique id.

Reply:
And we probably need to change the default sensor type to something else than `None`.

Reply:
Yeah, it being `NONE` is why I didn't do it.